README.md

```diff
@@ -1,2 +1,22 @@
 # password-generator
 Generate passwords given criteria
+
```

Comment:
As mentioned in the feedback, maybe add more to this. What was the criteria? What limitations are put on the user? How did you solve this?

Reply:
sorted